src/base/bittorrent/session.cpp

```diff
@@ -4779,6 +4866,29 @@ void Session::handleAlertsDroppedAlert(const lt::alerts_dropped_alert *p) const
 }
 #endif
 
+void Session::handleStorageMovedAlert(const lt::storage_moved_alert *p)
+{
+    if (m_moveStorageQueue.isEmpty()) return;
+
+    const TorrentHandle *torrent = m_torrents.value(p->handle.info_hash());
+    const MoveStorageJob &currentJob = m_moveStorageQueue.first();
+    if (currentJob.torrent != torrent) return;
+
+    const QString newPath {p->storage_path()};
+    handleMoveTorrentStorageJobFinished(newPath != currentJob.path ? tr("New path doesn't match a target path.") : QString{});
```

Comment:
I suppose you want `QString{}` -> `QString {}`

Reply:
Changed.